pom.xml

```diff
@@ -48,6 +48,7 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 
+		<jbtis.version>4.5.0.AM1-SNAPSHOT</jbtis.version>
```

Comment:
You still need it because
https://github.com/jbosstools/jbosstools-fuse/blob/master/targetplatform/pom.xml#L61
needs it.

Once that TP is no longer used, we can remove all reference to jbtis.